pkg/ddevapp/global_dotddev_assets/commands/host/phpmyadmin

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+## #ddev-generated: If you want to edit and own this file, remove this line.
+## Description: Prompt downloading phpMyAdmin if it is not installed.
+## Usage: phpmyadmin
+## Example: "ddev phpmyadmin"
+
+if [ "${DDEV_PROJECT_STATUS-running}" != "running" ] && [ -z "$no_recursion" ]; then
+  echo "Project ${DDEV_PROJECT} is not running, starting it"
+  ddev start
+  start_exit_code=$?
+  if [ $start_exit_code -ne 0 ]; then
+    exit $start_exit_code
+  fi
+  # run this script again, as the environment is updated after "ddev start"
+  no_recursion=true ddev "$(basename "$0")" "$@"
+  exit $?
+fi
+
+read -p "phpMyAdmin is not installed would you like to install? (yes/no): " answer
```

Comment:
```suggestion
read -p "phpMyAdmin is not installed would you like to install? (y/N): " answer
```

Reply:
With the case any combination will work, but I'm happy to change this.

Reply:
Yes, the capital letter just indicates what is default (which is quite common in the Linux world).